tests/test_rooms/test_room_filter.py

```diff
@@ -36,3 +38,10 @@ def test_filter_room_by_location_capacity(self):
             filter_rooms_by_location_capacity,
             filter_rooms_by_location_capacity_response
         )
+
+    def test_filter_room_by_wings_and_floors(self):
```

Comment:
Write tests for all the edge cases, i.e when there is no match and no results, when the filter parameter is not valid. Also, take care of the case sensitivity

Reply:
This has been worked on @Bonifase